src/share/content_renderer.js

```diff
@@ -11,7 +11,7 @@ function getChildrenList(note) {
         for (const childNote of note.getChildNotes()) {
             const li = document.createElement("li");
             const link = document.createElement("a");
-            link.appendChild(document.createTextNode(childNote.title));
+            link.appendChild(document.createTextNode(childNote.title + ` (${childNote.type})`));
```

Comment:
Includes note type in child note

Reply:
I'm not a fan of this. We're not doing this anywhere else so why here ...

Reply:
Good point